src/test/java/org/zalando/problem/spring/web/advice/io/MessageNotReadableAdviceTraitTest.java

```diff
@@ -26,6 +26,32 @@ public void missingRequestBody() throws Exception {
                 .andExpect(jsonPath("$.detail", containsString("request body is missing")));
     }
 
-    // TODO: jackson stuff tests

```

Comment:
the first one is a

> Specialized sub-class of {@link JsonMappingException}

and `Throwables.getRootCause` will return *this* if no cause was found. I think we should be safe here. I'll add two tests though.